src/components/SelectInput.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+<label for="select">
+    <span class="select__label">Category</span>
+    <select name="categories" class="select"
+    @change="$emit('input', $event.target.value)" >
+     <option selected value="All">All categories</option>
+     <option v-for="option in options" :key="option.id" :value="option.value">{{option.value}}</option>
+
+   </select>
+
+</label>
+
+    
+</template>
+
+<script>
+
+export default {
+    name: 'SelectInput',
+
+    data(){
+        return {
+            options: [
+                {id: 1, value:'Thriller'},
```

Comment:
Listing movie genres based on what you found in your movies list might be a cool and simple idea when you start working on your app, but will eventually cause problems. It isn't scalable at all. What if some new movie appears with a new genre? The application will break. It would be best to loop through all the movies and find all genres there. You can use traditional loops or some Array methods. It will make sure your genres list will always be up to date with existing movies. 😄